includes/functions.php

```diff
@@ -38,3 +38,15 @@ function affwp_cr_affiliate_display() {
 function affwp_cr_require_affiliate() {
 	return affiliate_wp()->settings->get( 'checkout_referrals_require_affiliate', false );
 }
+
+/**
+ * Get the affiliates sorting order.
+ * Provides a default for when the settings have not yet been saved.
```

Comment:
```suggestion
 * Gets the affiliates sorting order.
 *
 * Provides a default for when the settings have not yet been saved.
```
Use a third-person singular verb in the DocBlock summary and add a blank line between the summary and the description.